superset/assets/src/components/RefreshChartOverlay.jsx

```diff
@@ -25,24 +25,33 @@ const propTypes = {
   height: PropTypes.number.isRequired,
   width: PropTypes.number.isRequired,
   onQuery: PropTypes.func,
+  showOverlay: PropTypes.bool,
+  overlayLabel: PropTypes.string,
 };
 
+const DEFAULT_OVERLAY_MESSAGE = 'Placeholder to show overlay message';
+
 class RefreshChartOverlay extends React.PureComponent {
   render() {
     return (
       <div
         style={{ height: this.props.height, width: this.props.width }}
         className="explore-chart-overlay"
       >
-        <div>
+        {!this.props.showOverlay && <div>
           <Button
             className="refresh-overlay-btn"
             onClick={this.props.onQuery}
             bsStyle="primary"
           >
             {t('Run Query')}
           </Button>
-        </div>
+        </div>}
+        {this.props.showOverlay && <div>
+          <form className="overlay-chart">
+            <label className="overlay-label">{this.props.overlayLabel || DEFAULT_OVERLAY_MESSAGE}</label>
```

Comment:
by default DEFAULT_OVERLAY_MESSAGE should be shown in the control panel input

Reply:
As of now `TextControl` component doesn't support the default input value, so had to put the default here